lua/bufferline/bbye.lua

```diff
@@ -23,6 +23,14 @@
 -- For the full copy of the GNU Affero General Public License see:
 -- http://www.gnu.org/licenses.
 
+local buflisted = vim.fn.bufnr
+local bufnr = vim.fn.bufnr
+local command = vim.api.nvim_command
+local get_current_buf = vim.api.nvim_get_current_buf
+local reverse = require('bufferline.utils').reverse
+local set_current_buf = vim.api.nvim_set_current_buf
+local set_current_win = vim.api.nvim_set_current_win
```

Comment:
Nitpick but imports should be ordered by source, so `reverse` shouldn't be grouped with `vim.` imports.